runtime-interface/src/lib.rs

```diff
@@ -109,7 +116,8 @@ pub trait GearRI {
         gear_lazy_pages::get_released_pages()
     }
 
-    fn get_released_page_old_data(page: u32) -> Vec<u8> {
+    #[allow(clippy::result_unit_err)]
+    fn get_released_page_old_data(page: u32) -> Result<Vec<u8>, ()> {
```

Comment:
It is better to introduce custom unit-type for the error instead of using unit-type: https://rust-lang.github.io/api-guidelines/interoperability.html#error-types-are-meaningful-and-well-behaved-c-good-err .

Reply:
Ok